tendrl/commons/flows/create_cluster/ceph_help.py

```diff
@@ -222,12 +222,10 @@ def wait_for_task(task_id):
                 if resp["succeeded"]:
                     return
                 else:
-                    stderr = resp.get("stderr", "ceph-installer task_id %s failed and did not complete" % task_id)
-                    stdout = resp.get("stdout", "")
-                    raise FlowExecutionFailedError(dict(ceph_installer_task_id=task_id, ceph_installer_task_stdout=stdout,
-                                        ceph_installer_task_stderr=stderr))
+                    stderr = resp.get("stderr", "")
+                    raise FlowExecutionFailedError("ceph-installer task_id %s \
```

Comment:
Ah got the answer looking at the issue description. I am ok with this change.

May be small suggestion

```
raise FlowExecutionFailedError(
    "ceph-installer task_id %s failed and didnt complete. Error: %s" %
    (task_id, stderr)
)
```